code/modules/food_and_drinks/kitchen_machinery/deep_fryer.dm

```diff
@@ -93,6 +92,9 @@
 	var/input		//Thing that goes in
 	var/output		//Thing that comes out
 
+/datum/deepfryer_special/proc/validate(var/obj/item/I)
+	return(istype(I, input))
```

Comment:
Convention is to put a space after the return and the () is not required.
`return istype(...)` will do